src/r2mm/profiles/ProfilesClient.ts

```diff
@@ -0,0 +1,34 @@
+import Axios, { AxiosResponse } from 'axios';
+import R2Error from 'src/model/errors/R2Error';
+
+export default class ProfilesClient {
+    public static TSCreateAPI = 'https://thunderstore.io/api/experimental/legacyprofile/create/'
+    public static TSRetrieveAPI = (profileImportCode: string) => {return `https://thunderstore.io/api/experimental/legacyprofile/get/${profileImportCode}/`}
+    public static HerokuRetrieveAPI = (profileImportCode: string) => {return `https://r2modman-hastebin.herokuapp.com/raw/${profileImportCode}`}

```

Comment:
You don't need to use return here at all if you remove the `{}`. Semicolons missing